man_pages/dji_fpv_video_tool-play-video-with-osd.1

```diff
@@ -2,11 +2,11 @@
 .el .ds Aq '
 .TH play-video-with-osd 1  "play-video-with-osd " 
 .SH NAME
-play\-video\-with\-osd \- Plays a video using the MPV video player with OSD by overlaying transparent OSD video in real time
+play\-video\-with\-osd \- Play a DVR video using the MPV player, overlaying the transparent OSD video in real time
```

Comment:
It doesn't have to be a DVR / video from DJI FPV though, it just need to have a compatible resolution

Reply:
I wanted to distinguish "raw video from the air unit" from "video generated by this tool".

Reply:
I agree the name isn't great, do you have a better idea?

Reply:
In this case I think the previous version was fine. Just replacing "Plays" with "Play"

"Play a video using the MPV video player with OSD by overlaying a transparent OSD video in real time"

Reply:
Not sure we need to mention MPV. At least not in the command summary

"Play a video using with OSD by overlaying a transparent OSD video in real time"

Reply:
I agree, but was there already so I didn't remove it.